chainer/dataset/tabular/_concat.py

```diff
@@ -0,0 +1,110 @@
+import six
+
+from chainer.dataset.tabular import tabular_dataset
+
+
+class _Concat(tabular_dataset.TabularDataset):
+
+    def __init__(self, *datasets):
+        for dataset in datasets[1:]:
+            if not dataset.keys == datasets[0].keys:
```

Comment:
what is the intention?

Reply:
I just felt that the Python code `not a == b` is peculiar (I had some mental overhead on reading it), so I suggested a more straight expression. If you have a reason for choosing the original form, I would like you to leave the reason here to revisit later during maintenance.

Reply:
I don't have any reasons. I just wondered if any guideline exist.
I noticed `if not A == B` style is already used in chainer.
https://github.com/chainer/chainer/blob/master/chainer/functions/loss/sigmoid_cross_entropy.py#L46

Should I also fix this part?
https://github.com/chainer/chainer/blob/master/chainer/dataset/tabular/_slice.py#L64

Reply:
Oh, I didn't know there already exists the usage in sigmoid cross entropy function. I skimmed PEP8, where no mention is found on such a case. I made the suggestion because I felt it looks unusual to me and so I used a bit of time to imagine the special reason for writing an inequality test in this way here. From your response, I now understand it's just a style issue (writing in `not ==` way can look natural to other developers) which is not stated in our style guideline, so it's ok to leave it as is. Sorry for bothering you.